test/system/users_test.rb

```diff
@@ -0,0 +1,40 @@
+require 'application_system_test_case'
+
+class UsersTest < ApplicationSystemTestCase
+  setup do
+    @userone = users(:one)
+    @usertwo = users(:two)
+  end
+  test 'User index page' do
+    visit users_path
+    assert_selector 'a', text: @userone.name
+    assert_selector 'p', text: "Number of post #{@userone.posts_counter}"
+    assert 'avator.png', @userone.photo
+
+    assert_selector 'a', text: @usertwo.name
+    assert_selector 'p', text: "Number of post #{@usertwo.posts_counter}"
+    assert 'avator.png', @usertwo.photo
+    click_link @userone.name, match: :first
+    assert user_path(@userone)
+  end
```

Comment:
- [ ] Please make sure that you have a Spec file that only test `User` index page :pray: